mlflow/protos/unity_catalog_oss_messages.proto

```diff
@@ -21,6 +21,73 @@ message CreateRegisteredModel {
   }
 }
 
+message DeleteRegisteredModel {
+  // Required. Full name of the model.
+  optional string full_name_arg = 1 [(validate_required) = true, (scalapb.field).type = "FunctionFullName"];
```

Comment:
Let's change this to full_name in all places.  I had to fix my OSS PR on this regard for `reasons`.

Reply:
I also don't think you need the annotation list in any of the mlflow proto definitions.